src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -2358,6 +2358,16 @@ public async Task Parameter_DefaultValue()
                 MainDescription($"({FeaturesResources.parameter}) int param = 42"));
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        public async Task Lambda_Parameter_DefaultValue()
+        {
+            await TestInMethodAsync(
+@"(int param = 42) => {
+    return para$$m + 1;
+}",
+    MainDescription($"({FeaturesResources.parameter}) int param = 42"));
+        }
```

Comment:
test where hovering over `int`, `param` and `42` in the parameter section as well.